frontend/src/components/Common/FormModal.tsx

```diff
@@ -60,7 +61,8 @@ export interface FormModalProps<T extends FieldValues> {
   isOpen: boolean
   onClose: () => void
   title: string
-  fields: FieldDefinition<T>[]
+  fields?: FieldDefinition<T>[]
+  content: React.ReactNode
```

Comment:
@jakethrasher @Sergei-S-V Here I just added a second option for what's in the modal:
`fields` or `content` 

Each of these is used or not in the rendering of the modal.

There is definitely a smarter way to subclass this component and to use Typescript to encompass the two varieties form-only and content-only modals.